apps/greenhouse/src/components/core-apps/org-admin/components/plugins/styles.css

```diff
@@ -1,11 +1,13 @@
-// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
-// SPDX-License-Identifier: Apache-2.0
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
 
-/* Do not remove these tailwind directives. Without them styles won't work as expected */
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import "tailwindcss";
+@import "@cloudoperators/juno-ui-components/theme.css";
 
+/* If necessary, app styles can be added below */
+@layer base {
```

Comment:
I don't think it's necessary to add these app styles in `@layer base`. The layers are tailwind-created layers to be used for tailwind utility classes and they enforce an order in which styles are applied. this is important for utility classes. But here we dump custom styles that have nothing to do with tailwind. Let's not mix the two. (See Supernova example. Styles work just fine without being in `@layer base`).

Please try without `@layer base` and if it's correct that we don't need it, remove it everywhere.